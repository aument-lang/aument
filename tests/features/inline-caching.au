def f(a,b) {
    return a + b;
}

print f(10, 12);
print f(10, 12);
print f("A", "B");