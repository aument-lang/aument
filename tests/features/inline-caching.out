int;22
int;22
str;"AB"